tools/pythonpkg/tests/arrow/tpch/answers/q01.csv

```diff
@@ -0,0 +1,5 @@
+l_returnflag|l_linestatus|sum_qty|sum_base_price|sum_disc_price|sum_charge|avg_qty|avg_price|avg_disc|count_order
```

Comment:
Could we not duplicate queries and answers here but export them to Python from the `tpch`  extension?